HW12/src/Main.java

```diff
@@ -0,0 +1,66 @@
+import java.util.Arrays;
+
+public class Main {
```

Comment:
100 points for new tech awareness :)

P.S.
ChatGPT will soon replace all of us :)